src/www/ui/api/documentation/openapi.yaml

```diff
@@ -2156,6 +2172,12 @@ components:
               enum:
                 - OK
                 - ERROR
+    GroupAvailable:
+      type: object
+      properties:
+        available:
+          type: boolean
+          description: deletable group availability identifier
```

Comment:
IMO a more useful response will be a list of groups which the current user can delete (as it is returned by the `getDeletableAdminGroupMap()`).

By doing so, the UI can quickly remove groups which can not be deleted from a potential drop-down saving bunch of failing REST API calls.

As a suggestion, the response can look something like the following when the user can delete something:
```json
{
  "groups": [
      3,
      4,
      ...
    ]
}
```

Or like following when they can not delete any groups:
```json
{
  "groups": []
}
```

Reply:
Noted